assembly/pom.xml

```diff
@@ -174,63 +174,19 @@
                 <descriptor>
                   mvn:org.apache.jclouds.karaf/jclouds-karaf/${jclouds.karaf.version}/xml/features
                 </descriptor>
+                <descriptor>
+                  mvn:org.apache.jclouds.karaf/jclouds-karaf-labs/${jclouds.karaf.version}/xml/features
+                </descriptor>
               </descriptors>
               <features>
                 <feature>config</feature>
-                <feature>jclouds</feature>
-                <feature>jclouds-commands</feature>
-                <feature>jclouds-compute</feature>
-                <feature>jclouds-guice</feature>
```

Comment:
Do we need to keep these? They are not directly included by the macro features, from what I can see?

Even if they don't _need_ to be included, would it be clearer to keep them?